catalog/ui/catalog-ui-search/src/main/webapp/component/system-usage/system-usage.less

```diff
@@ -10,6 +10,10 @@
         border: none;
     }
 
+     ul {
```

Comment:
This won't have any effect on the iframe though.  

If there's specific styling wanted for a message, that's already possible by including it in a style tag in the message itself.